server/paymentProviders/stripe/creditcard.js

```diff
@@ -175,13 +175,6 @@ const createChargeAndTransactions = async (hostStripeAccount, { order, hostStrip
     throw new Error(UNKNOWN_ERROR_MSG);
   }
 
-  // Success: delete reference to paymentIntent
-  // TODO: paymentIntent should be later removed anyway on success, see subscriptions.js and orders.js
-  if (order.data.paymentIntent) {
-    delete order.data.paymentIntent;
-    await order.update({ data: order.data });
-  }
```

Comment:
Why are we removing that code now?

Reply:
There was a note saying it should be taken care of in orders.js and subscriptions.js. 

Subscriptions.js doesn't exist anymore, and in orders.js (v1 mutations) we call `libPayments.executeOrder` in both `createOrder` and `confirmOrder`, which has a line to take care of deleting `paymentIntent`: https://github.com/opencollective/opencollective-api/blob/c831017ae342df590dab217c3df5d07a0ed39bd7/server/lib/payments.js#L329

So it's not necessary anymore.